tests/integration/test_charm.py

```diff
@@ -25,10 +55,17 @@ def fix_queryselector(elems):
 
 
 @pytest.fixture()
-def driver(request):
-    status = yaml.safe_load(check_output(["juju", "status", "--format=yaml"]))
+async def driver(request, ops_test):
+    status = yaml.safe_load(
```

Comment:
As long as we're using `application.get_config()` down below, will that give us the endpoint instead of running `juju status` ourselves here?

Reply:
This is a very annoying part of libjuju, there isn't a way of getting the address field.

This[bug](https://github.com/juju/python-libjuju/issues/551) might be a related but there probably should be a more specific one.

Reply:
Filed a [bug](https://github.com/juju/python-libjuju/issues/559) for it.